hawkbit-runtime/hawkbit-update-server/src/main/resources/application-cloudsandbox.properties

```diff
@@ -32,5 +32,11 @@ hawkbit.artifact.url.protocols.md5sum-http.ref=${hawkbit.artifact.url.protocols.
 security.user.name=demo
 security.user.password=demo
 
-hawkbit.server.ui.demo.password=${security.user.name}
 hawkbit.server.ui.demo.user=${security.user.name}
+hawkbit.server.ui.demo.password=${security.user.name}
+hawkbit.server.ui.demo.disclaimer=<small>By signing in, you consent that we store the following data for up to one week: \
+  <ul><li><b>IP-Address:</b> Your client's IP-Address, as well as, the IP-Address of any device you connect to the \
+  application are stored for the purpose of misuse prevention.</li></ul>\
+  <p>You are not permitted to store any kind of personal data in this application, since this is a shared account.</p> \
+  <p><b>Credentials:</b> <code>demo:demo</code></p></small>
```

Comment:
Would it make sense to pre-fill the input fields for username and password with the demo credentials?